WISE-Community/WISE-Client
# build(Draw Tool): Use modified draw tool

## Changes

Changed dependency to use our modified draw tool.

## Test

Run ```npm ci```

- Make sure the draw tool now opens tool submenus the second time you click a tool
- When the stamp tool is clicked the first time, and no stamp has previously been selected, it should display the stamp submenu
- Make sure each tool icon remains highlighted after you select an item in the tool submenu

Closes #1494
